frontend/freebase2wikidata.js

```diff
@@ -260,7 +260,56 @@ $(function() {
     var title = mw.config.get('wgTitle');
     return qidRegEx.test(title) ? title : false;
   }
+  
+ function generateNav() {
+    $('#p-ps-navigation').remove();
+    $('#mw-panel').append('<div class="portal" role="navigation" id="p-ps-navigation" aria-labelledby="p-ps-navigation-label"><h3 id="p-ps-navigation-label">PS navigation tool</h3></div>');
+    $('#p-ps-navigation').append('<div class="body"><ul id="p-ps-nav-list"></ul></div>');
+    
+    var b = [];
+    var c = [];
+    
+    $('.wikibase-statementgroupview').each(function(index, item) {
+      var a = $(item).find('.new-object')[0];
+      if (a) {
+        b.push($(item).find('.wikibase-statementgroupview-property-label').find('a'));
+        c.push($(a).find('.valueview-instaticmode')[0]);
+      }
+    });
+
+    for (var i = 0; i < b.length; i++) {
+      var text = b[i].text();
+      $('#p-ps-nav-list').append('<li id="n-ps-anchor' + text.replace(/\s+/g, '') + '"><a href="#" title="move to ' + text + '">' + text + '</a></li>');
+      $('#n-ps-anchor' + text.replace(/\s+/g, '')).click({
+        counter: i
+      }, function(e) {
+        e.preventDefault();
+        c[e.data.counter].scrollIntoView();
+      });
+    }
+
+    $(function() {
```

Comment:
```$(function() {})``` wrapping is not needed here, however it would be nice if this chunk will be in a separate function (accepting ```$sidebar``` as an argument, for example)